src/rust/engine/dep_inference/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-version = "0.0.1"
+version = "0.0.1.1"
```

Comment:
I manually bumped this to `0.0.1.1` so we don't collide with `0.0.2` from `main`

Reply:
Oh so apparently that isn't kosher... ugh

Reply:
https://github.com/pantsbuild/pants/pull/19790